strawberry/types/types/argument.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional, Type, TypeVar
+
+from .object import StrawberryObject
+from .type import StrawberryType
+
+T = TypeVar("T", bound=StrawberryObject)
+
+
+class StrawberryArgument(StrawberryType[T]):
+    def __init__(self, name: str, type_: Type[T], *,
+                 description: Optional[str],
+                 # TODO: Can we use None, or do we want fields.NO_DEFAULT_VALUE?
```

Comment:
I think this is similar to default values for fields, we can’t really use None by default :)